src/main/java/com/dreamteam/database/GUI.java

```diff
@@ -0,0 +1,364 @@
+                panel2.add(product);
+                panel2.add(quantityLabel);
+                panel2.add(quantity);
+                panel2.add(wholesaleLabel);
+                panel2.add(wholesale);
+                panel2.add(salesLabel);
+                panel2.add(sales);
+                panel2.add(supplierLabel);
+                panel2.add(supplier);
+                panel2.add(submit);
+
+                newGUI("CREATE");
+
+                break;
+
+            case "READ":
+                product = new JTextArea();
+                panel2.add(productLabel);
+                panel2.add(product);
+                submit = new JButton("SUBMIT");
+                submit.addActionListener(this::readSubmit);
+                panel2.add(submit);
+                newGUI("READ");
+
+                break;
+
+            case "UPDATE":
+                EnumSet<Options> OPTIONS = EnumSet.of(Options.QUANTITY, Options.CAPACITY, Options.WHOLESALE_COST, Options.SALE_PRICE, Options.SUPPLIER, Options.DONE);
+                ArrayList<JButton> buttonList = new ArrayList<>();
+                for(Options option : OPTIONS){
+                    buttonList.add(new JButton(String.valueOf(option)));
+                }
+                for(int i = 0; i < buttonList.size(); i++){
+                    panel2.add(buttonList.get(i));
+                }
+                newGUI("UPDATE");
+
+                break;
+
+            case "DELETE":
+                product = new JTextArea();
+                panel2.add(productLabel);
+                panel2.add(product);
+                submit = new JButton("SUBMIT");
+                submit.addActionListener(this::deleteSubmit);
+                panel2.add(submit);
+                newGUI("DELETE");
+
+                break;
+
+            case "PROCESS_ORDERS":
+                label.setText("Press button to process orders.");
+                submit = new JButton("PROCESS");
+                submit.addActionListener(this::processSubmit);
+                panel2.add(label);
+                panel2.add(submit);
+                newGUI("PROCESS ORDERS");
+
+                break;
+
+            case "REPORTS":
+                JLabel reportsLabel = new JLabel("For which date would you like reports for?");
+                date = new JTextArea();
+                label.setText("You can also view a graph report of company assets.");
+                view = new JButton("VIEW GRAPH REPORT");
+                submit = new JButton("GENERATE ORDER REPORT");
+                submit.addActionListener(this::reportSubmit);
+                view.addActionListener(this::viewSubmit);
+                panel2.add(reportsLabel);
+                panel2.add(date);
+                panel2.add(submit);
+                panel2.add(label);
+                panel2.add(view);
+                newGUI("REPORTS");
+
+                break;
+
+            case "QUIT":
+                System.exit(0);
```

Comment:
Change to frame.dispose() and frame2.dispose() to release gui resources without forcing a shutdown of the program.